ophyd/positioner.py

```diff
@@ -62,7 +54,7 @@ def low_limit(self):
     def high_limit(self):
         return self.limits[1]
 
-    def move(self, position, wait=True, moved_cb=None, timeout=30.0):
+    def move(self, position, moved_cb=None, timeout=30.0):
```

Comment:
Waiting can only happen post-return from `PositionerBase`, so all subclass `move` methods should accept it but `PositionerBase` never sees it